docs/v3/core/index.rst

```diff
@@ -131,10 +132,10 @@ with implementation B.
 Therefore, data is only marked with the respective major version, unknown
 features are auto-discovered via the metadata document.
 
-Notably, this excludes extension points such as codecs, data types, chunk grids
+Notably, this excludes extensions such as codecs, data types, chunk grids
 and storage transformers from the compatibility of the core specification, as
-well as store support. However, versioned extension points and stores are also
-expected to follow this stability policy.
+well as store support. However, extensions and stores are also RECOMMENDED to 
+follow this stability policy.
```

Comment:
this applies to the original text, but as long as we are editing it: what exactly does this statement mean? Why are we talking about versioning stores here? And why do we care if extensions use a specific versioning scheme? I think a motivation is necessary, because otherwise it's not clear.